### PR TITLE
Removed the match/certs step in the makefile + a readme update

### PR DESCRIPTION
**Pull request by mennenia**
As the title says.

Readme contains some info I found useful getting setup. 

@sarahscott @orta

### CODE REVIEW

**Comment by mennenia** [comment]
Test comment for slack bot

**Comment by mennenia** [comment]
Let's try another test comment...

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Error
    ------------- | ------------
        :no_entry_sign: | No CHANGELOG changes made
    

  

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Comment by orta** [comment]
Testing

File: README.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ You will have a running version of the Artsy app by hitting `Build > Run`.
 
 ### Work at Artsy?
 
-Instead of `make oss` in the above, run `make artsy` to set up [spacecommander](https://github.com/square/spacecommander).
+- Instead of `make oss` in the above, run `make artsy` to set up [spacecommander](https://github.com/square/spacecommander).
+- Set an environment variable of `ARTSY_STAFF_MEMBER` to `true`.
```
Comment (sarahscott) [inline README.md]: @orta this is how I got Artsy fonts to work for myself and Maxim, but I'm not sure if it was always this way

Comment (orta) [inline README.md]: We should get that in the [mobile playbook](https://github.com/artsy/mobile/blob/master/playbook.md)

**Comment by orta** [comment]
The issue with danger is known, https://github.com/KrauseFx/danger/issues/45

File: README.md (hunk 1 above)
Comment (mennenia) [inline README.md]: should I then take it out of the readme here? I'll add it there also

Comment (orta) [inline README.md]: I'm chill about removing, but I'm sure on adding to playbook

**Pull request merged**